apps/web/src/utils/experienceUtils.tsx

```diff
@@ -260,6 +329,19 @@ export const formValuesToSubmitData = (
     experienceTitle,
     experienceDescription,
     currentRole,
+    employmentCategory,
+    extSizeOfOrganization,
+    extRoleSeniority,
+    department: departmentId,
+    govEmploymentType,
+    govPositionType,
+    govContractorRoleSeniority,
+    govContractorType,
+    contractorFirmAgencyName,
+    classificationLevel: classificationId,
+    cafEmploymentType,
+    cafForce,
+    cafRank,
   } = data;
 
   const newEndDate = !currentRole && endDate ? endDate : null;
```

Comment:
Will need to update this, given that currentRole is true and endDate is truthy is now a possibility with the expected end date

![image](https://github.com/user-attachments/assets/906d4224-40d5-404c-af5f-aca18738b64d)

Reply:
Thank you! I was looking for that! 
Updated endDate logic in [ec0e17b](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12208/commits/ec0e17b104d4fb5bd6d0b993a9fb9f43b0f8e233)